packages/dd-trace/src/dcitm.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const dc = require('diagnostics_channel')
+
+const CHANNEL_PREFIX = 'dd-trace:bundledModuleLoadStart'
+
+if (!dc.subscribe) {
```

Comment:
Whoops, these shouldn't be needed anymore, I'll remove.

Reply:
Doesn't look like they were removed?

Reply:
They were still needed after all